src/constants.ts

```diff
@@ -45,6 +45,8 @@ export const CONSTANTS = {
       "error.debuggingSessionInProgress",
       "[ERROR] A debugging session is currently in progress, please stop it before running your code. \n"
     ),
+    DEPENDENCY_DOWNLOAD_ERROR:
+      "Package downloads failed. Some functionality may not work. ",
```

Comment:
Maybe we can hint the user how to address this? Something like: "Try restarting the simulator or check out [link to project install.md]".

Reply:
Great idea! Implementing that now